src/encrypter.ts

```diff
@@ -72,7 +72,10 @@ export class Encrypter {
     if (internalClient == null) {
       const clonedOptions: MongoClientOptions = {};
 
-      for (const key of Object.keys(options)) {
+      for (const key of [
+        ...Object.getOwnPropertyNames(options),
+        ...Object.getOwnPropertySymbols(options)
+      ] as string[]) {
```

Comment:
This is for passing through `[Symbol.for('@@mdb.skipPingOnConnect')]: true` to the AutoEncrypter internal client 🙂